src/NuGetGallery/Controllers/ODataV1FeedController.cs

```
@@ -185,13 +205,18 @@ private async Task<IHttpActionResult> GetCore(ODataQueryOptions<V1FeedPackage> o
                     customQuery = true;
                 }
             }
-            catch (Exception ex)
+            catch (Exception ex) when (isNonHijackEnabled)
```

I'm not sure I understand this. Do we want to throw an exception if the flag is disabled? Who catches the exception? Maybe add a comment if this is intentional.

---

Also, do you know if this exception is covered by a test?

---

If our only dependency fails (the case when the FF is disabled), we should 500.

---

Ahh I see, that makes sense. I would add a quick comment for folks like me that aren't very fast on the uptake 😂

---

Done. More comments are good :)